Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM rust:1.54.0-buster as planner
+WORKDIR /app
+RUN curl https://getsubstrate.io -sSf | bash -s \
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
+COPY . .
+RUN cargo build --release \
+    && mv target/release/node-template /usr/local/bin/supra-dht \
+    && mv /usr/local/cargo/bin/sub* /usr/local/bin/ \
+    && rm -rf target \
+    && rm -rf /root/.cargo/registry/ \
+    && rm -rf /tmp && rm -rf /usr/local/rustup && rm -rf /usr/local/cargo
+
+FROM rust:1.54.0-buster as runtime
+COPY --from=planner /usr/local/bin/supra-dht /usr/local/bin/
+COPY --from=planner /usr/local/bin/sub* /usr/local/bin/
+EXPOSE 30333 9933 9944
+CMD ["supra-dht", \
```

Comment:
Use entry point instead so that the docker can accept commands just as if the client was installed locally in your bin.

Reply:
You mean, just let it be - `ENTRYPOINT ["supra"]` ?